File: Solution in C++/Assignment_02/Task14.cpp
#include<iostream>
using namespace std;

int main(){
    int s,t;
    float v;
    cin>>s>>t;
    v=(s/1000)/(t/3600);
    cout<<v<<endl;
    if(v<60){cout<<"Too slow. Needs more changes.";}
    else if(v<=90){cout<<"Velocity is okay. The car is ready!";}
    else{cout<<"Too fast. Only few charges should saffice";}
}

File: Solution by Python/Assignment_02/Task14.py
m=int(input())
n=int(input())
v=(m/1000)/(n/3600)
print(v,"km/h")
if v<0:
    print("Invalid")
elif v<60:
    print("Too slow. Needs more changes")
elif v<=60 and v>=90:
    print("Velocity is okay. The car is ready!")
else:
    print("Too fast. Only a few changes should suffice.")